.circleci/config.yml

```diff
@@ -9,8 +9,7 @@ jobs:
       - checkout
       - node/with-cache:
           steps:
-            - run: npm install yarn
-            - run: yarn install
+            - run: npm install
```

Comment:
I don't think this change is meaningful, but Circle updated this as their default config for a node app.